bin/fuel-core/src/cli/rollback.rs

```diff
@@ -19,6 +23,17 @@ pub struct Command {
     )]
     pub database_path: PathBuf,
 
+    /// Defines a specific number of file descriptors that RocksDB can use.
+    ///
+    /// If defined as -1 no limit will be applied and will use the OS limits.
+    /// If not defined the system default divided by two is used.
```

Comment:
What's the reason for using `-1` to mean no limit as opposed to take an `Option<u32>`?

Reply:
It is the default value used by RocksDB for no limit that's why they are a i32.

Reply:
And we want to have different behaviour between : 
- No defined value which will be using OS divided by two
- and max value of the system which is -1

Reply:
Alright, fair enough.

Reply:
I agree that this limit should generally be `Option<u32>`, but this can only represent two states, not the three we need. Let's keep it like that for the sake of simplicity. But we should maybe sanitize this value and not allow -2, -3, etc.?

Reply:
RocksDB doesn't seems to consider these values as error : https://github.com/FuelLabs/fuel-core/pull/2386#discussion_r1814657998

But we can do. I don't think it's very useful because it doesn't creates any problem and the only reason to tryy a -2 is to tryy to make a problem.